### ISSUE TITLE
Support message attributes

### ISSUE THREAD

**Issue by runningman84** [DataReply/alertmanager-sns-forwarder#24]
I would like to add some message attirbutes to my sns messages, this is supported by SNS:
https://docs.aws.amazon.com/sns/latest/dg/sns-tutorial-publish-message-with-attributes.html

My usecase is that we have a central notification topic for multiple clusters/customers. I would add static attirbutes like cluster_name, customer_name and so on... 

I could also see a use for dynamic attributes like alert_status, alert_receiver and alert_external_url

This would allow us to filer these message in sns too...

**Comment by tcdev00** [comment]
Yeah, that's pretty similar to my use case too, would be really good to define the MessageAttributes from the forwarder.
